### ISSUE TITLE
MixFormer that is trained on GOT10K without pretrained weights  seems collapse?

### ISSUE THREAD

**Issue by zorrocai** [MCG-NJU/MixFormer#32]
Hi, we ran the miformer experiments without pretrained cvt weights on got10k, using the default configuration. The results show that after 200 epochs, the AO on the got10k test set is only 0.096. It is not clear where the problem occurred. Do you have valuable advice?
